storage/postgres/admin_storage.go

```diff
@@ -188,11 +201,63 @@ func (t *adminTX) GetTree(ctx context.Context, treeID int64) (*trillian.Tree, er
 }
 
 func (t *adminTX) ListTrees(ctx context.Context, includeDeleted bool) ([]*trillian.Tree, error) {
-	return nil, errNotImplemented
+	var query string
+	if includeDeleted {
+		query = selectTrees
+	} else {
+		query = selectNonDeletedTrees
+	}
+
+	stmt, err := t.tx.PrepareContext(ctx, query)
+	if err != nil {
+		return nil, err
+	}
+	defer stmt.Close()
```

Comment:
nit: It'd be nice if the whitespace were consistent between methods. In `ListTreeIDs()`, for example, you have a blank line here.